src/java.desktop/share/classes/java/awt/color/ColorSpace.java

```diff
@@ -447,60 +451,37 @@ public int getNumComponents() {
     /**
      * Returns the name of the component given the component index.
      *
-     * @param  idx the component index
+     * @param  component the component index
```

Comment:
All other methods use the "component" as a name of the component index.

Reply:
I guess it will require a CSR request.

Reply:
nope, it is just renaming of the parameter